src/api/status/public/index.html

```diff
@@ -36,8 +36,9 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Round" rel="stylesheet" />
     <!-- CSS Files -->
     <link id="pagestyle" href="assets/css/material-dashboard.css?v=3.0.0" rel="stylesheet" />
+    <link id="" />
+    <link rel="stylesheet" type="text/css" href="./style.css" />
```

Comment:
We're discussing how to standardize where we want to put our style overrides for Telescope.  cc @Andrewnt219, @menghif, for thoughts.

Reply:
I think we can have a new file in `assets/css` and call it something like `telescope-overrides.css`

Reply:
The css file is already in the public folder not in the asset folder